libraries/resource_providers/api2000/c7000/storage_system_provider.rb

```diff
@@ -0,0 +1,19 @@
+# Licensed under the Apache License, Version 2.0 (the "License");
```

Comment:
first line in copyright seems like missing

Reply:
Done